src/lib.rs

```diff
@@ -266,8 +267,23 @@ impl<'a> Executor<'a> {
     fn schedule(&self) -> impl Fn(Runnable) + Send + Sync + 'static {
         let state = self.state().clone();
 
-        // TODO: If possible, push into the current local queue and notify the ticker.
-        move |runnable| {
+        move |mut runnable| {
+            // If possible, push into the current local queue and notify the ticker.
+            if let Some(local) = state.local_queue.get() {
```

Comment:
Here's another issue I've realized:

- `ex.run()` is called, initializing `state.local`.
- Eventually `ex.run()` ends. It drains `state.local`.
- The same thread later calls `spawn` and pushes a task to the local queue.
- However no runner is polling that queue.
- No thread pulls that task.

I don't think this is an issue since another thread will come along to steal it. However it should probably be commented here that that happens.

Reply:
Done.